build.gradle

```diff
@@ -506,22 +505,30 @@ task testDocker {
   }
 }
 
-task uploadDocker {
-  dependsOn([distDocker])
-  def dockerBuildVersion = "${rootProject.version}".replace('+', '-')
-  def architecture = System.getenv('architecture')
-  def platform = System.getenv('platform')
-
+def dockerTagPrefixes(String dockerBuildVersion) {
   def versionPrefixes = [dockerBuildVersion]
   if (project.hasProperty('branch') && project.property('branch') == 'master') {
     versionPrefixes.add('develop')
   }
 
-  if (!isDevelopBuild) {
+  // add `latest` tag for non-develop and non-rc versions
+  if (!rootProject.version.contains('develop') && !rootProject.version.toLowerCase().contains('-rc')) {
     versionPrefixes.add('latest')
+    // when docker build version is x.y.[a,b..] we would also like to use tag x.y
```

Comment:
Guess this doesn't hurt, but probably don't need this either

Reply:
It was already part of existing logic, I simply added the comment for readability.